packages/app/components/camera/camera-buttons.tsx

```diff
@@ -5,8 +5,8 @@ import { Flip, Close, Check } from "@showtime-xyz/universal.icon";
 import { PressableScale } from "@showtime-xyz/universal.pressable-scale";
 import { View } from "@showtime-xyz/universal.view";
 
-import { FilePickerResolveValue } from "design-system/file-picker";
-import { ImagePickerButton } from "design-system/image-picker";
+import { FilePickerResolveValue } from "app/lib/file-picker";
```

Comment:
move the `file-picker` and `image-picker` folders to `app/lib/*`, I think It should not be in the ·design-system·